### PR TITLE
docs: conventional commits

### PR DESCRIPTION
**Pull request by ElTimuro**

### CODE REVIEW

**Comment by ElTimuro** [comment]
@Mythie

**Comment by ElTimuro** [comment]
@dephraiim

**Comment by dephraiim** [comment]
Adding it to the docs isn't enough, someone may miss it. Why don't we enforce it with git hooks?

**Comment by Mythie** [comment]
> Adding it to the docs isn't enough, someone may miss it. Why don't we enforce it with git hooks?
> 
> https://github.com/commitizen/cz-cli This is a good starting point

Yep precisely, I've been meaning to do something with Husky and Commitizen

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,7 +23,7 @@ The development branch is <code>main</code>. All pull request should be made aga
 - Create a new branch (include the issue id and something readable):
 
   ```sh
-  git checkout -b doc-999-my-feature-or-fix
+  git checkout -b doc-999-feat-somefeature-that-rocks
```
Comment (Mythie) [inline CONTRIBUTING.md:26]: I prefer `feat/doc-999-some-feature-that-rocks`

Comment (ElTimuro) [reply CONTRIBUTING.md:26]: fine for me

Comment (ElTimuro) [reply CONTRIBUTING.md]: @Mythie merged main. we stop using the slack slug for updating? either way ready to merge

Comment (Mythie) [reply CONTRIBUTING.md]: We're definitely using `documen.so/slack` not sure why you've gotten an old one here. Please update as part of this 😄

Comment (ElTimuro) [reply CONTRIBUTING.md]: zoink

Comment (ElTimuro) [reply CONTRIBUTING.md]: boing

**Review by Mythie — APPROVED** [review]

**Comment by ElTimuro** [comment]
@Mythie you said sth. about enforcing, which is great. Can I merge or should we wait to enforce? happy to wait and keep this as a reminder

**Comment by Mythie** [comment]
> @Mythie you said sth. about enforcing, which is great. Can I merge or should we wait to enforce? happy to wait and keep this as a reminder

No need, I'll make this the case shortly

**Pull request merged**